### ISSUE TITLE
[Feature request] Create multiple files at once

### ISSUE THREAD

**Issue by TSSReal** [patbenatar/vscode-advanced-new-file#114]
Instead of doing ctrl + n and creating one file to then repeat the same process when you could implement something to create multiple files at once. For example typing in dog.html, cat.css

**Comment by Archetipo95** [comment]
Yep I would like to use it, maybe something like comma separated files:
`MyNewFile.vue, MyNewFile.stories.ts, MyNewFile.props.ts`

**Comment by roryfahy** [comment]
I would also really like this functionality, I wonder how hard it would be to make these mods.  I dont typically program js but might be able to take a look at it if someone could point me in the right direction

**Comment by louiss0** [comment]
I would also like the feature. without that this extension is not better than using the terminal at all.
